Atomic/util.py

```diff
@@ -842,10 +842,16 @@ def wrapper(*args, **kwargs):
 
 
 class InstallData(object):
-    if not os.path.exists(ATOMIC_INSTALL_JSON):
-        open(ATOMIC_INSTALL_JSON, 'a').close()
 
-    install_file_handle = open(ATOMIC_INSTALL_JSON, 'r')
+    @staticmethod
+    def get_install_file_handle():
+        if not os.path.exists(ATOMIC_VAR_LIB):
+            os.makedirs(ATOMIC_VAR_LIB)
+
+        if not os.path.exists(ATOMIC_INSTALL_JSON):
+            open(ATOMIC_INSTALL_JSON, 'a').close()
```

Comment:
I don't understand why we create an empty file explicitly.  It's not valid JSON.  Why not simply have "absence of file" = empty?  Basically we don't explicitly write it when we're just opening for *read*.